var/spack/repos/builtin/packages/qt-base/package.py

```diff
@@ -65,9 +65,12 @@ def cmake_args(self):
             if re_qt.match(dep.name):
                 qt_prefix_path.append(self.spec[dep.name].prefix)
 
-        # Now append all qt-* dependency prefixex into a prefix path
+        # Now append all qt-* dependency prefixes into a prefix path
         args.append(self.define("QT_ADDITIONAL_PACKAGES_PREFIX_PATH", ":".join(qt_prefix_path)))
 
+        # Add all dependency prefix/lib directories to RPATH
+        args.append(self.define("QT_EXTRA_RPATHS", ":".join(p + "/lib" for p in qt_prefix_path)))
```

Comment:
Not quite happy with this one:
- explicit `lib` instead of `lib64`
- explicit slash won't work on windows

Reply:
These are cmake lists? Then they should just be python lists as a value, no need for `":".join`

Reply:
```suggestion
        qt_extra_rpaths = [p.lib for p in qt_prefix_path] + [p.lib64 for p in qt_prefix_path]
        args.append(self.define("QT_EXTRA_RPATHS", qt_extra_rpaths))
```